dev.yml

```diff
@@ -2,7 +2,7 @@
 name: recordstore
 
 up:
-  - ruby: 2.5.0
+  - ruby: 2.5.7
```

Comment:
Just curious - what motivated this ruby upgrade?

Reply:
was just testing this to see if it's somehow related to the failing test(s).
they don't fail locally :-(

Reply:
reverted this ruby version bump

the issue was the api host name changed with a google-cloud-dns gem version

(I've updated cassettes and locked the gem version)